Spec/Tests/RestClientTests.swift

```diff
@@ -1533,46 +1533,32 @@ class RestClientTests: XCTestCase {
         testStoresSuccessfulFallbackHostAsDefaultHost(.hostInternalError(code: 501))
     }
 
-    func beforeEach__RestClient__Host_Fallback__should_store_successful_fallback_host_as_default_host__should_restore_default_primary_host_after_fallbackRetryTimeout_expired() {
-        ARTDefault.setFallbackRetryTimeout(1.0)
-    }
-
     func test__080__RestClient__Host_Fallback__should_store_successful_fallback_host_as_default_host__should_restore_default_primary_host_after_fallbackRetryTimeout_expired___hostUnreachable() {
-        beforeEach__RestClient__Host_Fallback__should_store_successful_fallback_host_as_default_host__should_restore_default_primary_host_after_fallbackRetryTimeout_expired()
-
+        
```

Comment:
(minor) I don't think we need this empty line and the ones in the subsequent methods?

Reply:
82868c34

Reply:
82868c34c35070fe4c2ff8881255e4a3703d2f4e